.github/workflows/test.yml

```diff
@@ -47,13 +47,27 @@ jobs:
     name: Linux x86-64
     runs-on: solang-ubuntu-latest
     container: ghcr.io/hyperledger/solang-llvm:ci-5
+    env:
+      RUSTFLAGS: -C instrument-coverage -C llvm-args=--instrprof-atomic-counter-update-all --cfg=coverage --cfg=trybuild_no_target
+      LLVM_PROFILE_FILE: ${{ github.workspace }}/target/solang-%p-%10m.profraw
+      CARGO_INCREMENTAL: 0
+      CARGO_LLVM_COV: 1
+      CARGO_LLVM_COV_TARGET_DIR: ${{ github.workspace }}/target
```

Comment:
Rather than repeating the env for each job, the env can be set on the entire workflow: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env